src/main/webapp/templates/QuestionChoiceItemView.html

```diff
@@ -1,20 +1,26 @@
 
```

Comment:
Not sure why this file needed to change??

Reply:
This is the file that very strangely appeared changed in my working tree!!! I did not make these changes and I don't understand where they are coming from as we discussed. Will try restarting these changes as you suggested. I'll ping you if this happens again after I reload.